src/components/ui/popover/popover.component.tsx

```diff
@@ -36,6 +36,7 @@ type PopoverModalProps = Omit<ModalProps, ' children'>;
 
 export interface PopoverProps extends PopoverViewProps, PopoverModalProps, RNModalProps {
   children?: React.ReactElement;
+  placement: PopoverPlacement;
```

Comment:
`PopoverViewProps` already includes placement prop, but it's optional there. is this override intentional?

Reply:
yes, it is